golang/cosmos/NOTICE

```diff
@@ -0,0 +1,5 @@
+Agoric SDK
+Copyright 2018-2022 Agoric Systems Operating Company
+
+This product includes software developed at
+Agoric Systems Operating Company (https://agoric.com/).
```

Comment:
This directory might already also include code copyright the cosmos teams.

Reply:
Why do we need a `NOTICE` file? [Apache guidance](https://infra.apache.org/licensing-howto.html) says,
> The `NOTICE` file … is required by [ASF policy](https://www.apache.org/legal/src-headers.html#notice).

We're using APL not part of ASF.

Reply:
AFAICT, 99% of code in `golang/` is our code that uses the Cosmos SDK, not code copied in from Cosmos SDK.  

The exception is `golang/cosmos/third_party/proto` which I'll give credit to "The GoGo Authors" and "Google Inc." in the NOTICE file.